GARTH-NIX-OLD-KINGDOM-BLOODLINES.md

```
@@ -0,0 +1,194 @@
+
+### [[TROUBLE-BUBBLE]]
+- **Connection**: The **Disreputable Dog is Kibeth**, one of the **Nine Bright Shiners** (specifically, the Walker). Kibeth retained consciousness and appears as a dog
+- **Parallel**: Mr. Trouble-Bubble Curtis Gonzalez-Finney the Doodle represents a similar concept of immense power in an unexpected, unassuming form
+- **Canon Source**: *The Old Kingdom* Wikipedia: "the Disreputable Dog (a self-identified remnant of the spirit 'Kibeth')"
+- **Fun Note**: Both subvert expectations of what a powerful cosmic being "should" look like
+
+---
+
+## Wikilinks Network
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)